public/editor/stylesheets/editor.less

```diff
@@ -714,6 +714,7 @@ body {
   .nav-container {
     min-width: 400px;
     padding: 0 16px 0 20px;
+	white-space: nowrap;
```

Comment:
please use the same indentation as the rest of the file (so: indentation using spaces, not tabs)